src/Donations/index.tsx

```diff
@@ -140,6 +144,19 @@ function DonationInfo() {
     const callbackUrl = router.query.callback_url;
     router.push(`${callbackUrl ? callbackUrl : "/"}`);
   };
+  const [anchorEl, setAnchorEl] = React.useState<HTMLElement | null>(null);
```

Comment:
Naming is important. Naming this as `anchorEl` instead of `anchorElement` makes the code less readable.